src/main/java/org/broadinstitute/hellbender/metrics/MetricsArgumentCollection.java

```diff
@@ -1,19 +1,20 @@
 package org.broadinstitute.hellbender.metrics;
 
-
 import org.broadinstitute.hellbender.cmdline.Argument;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
 
 import java.io.File;
+import java.io.Serializable;
 
 /**
  * Base class for defining a set of metrics collector arguments. The members should be
  * instantiable as command line arguments.
  */
-public class MetricsArgumentCollection {
```

Comment:
Should this be an abstract class?  Will anyone want to instantiate one of the directly?